src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingTemplate.cs

```diff
@@ -84,54 +87,54 @@ public static BindingTemplate FromString(string input, bool ignoreCase = false)
         /// <returns>Resolved string if succeeded.</returns>
         /// <exception cref="InvalidOperationException">Thrown when required parameter value is not available.
         /// </exception>
-        public string Bind(IReadOnlyDictionary<string, string> parameters)
+        public string Bind(IReadOnlyDictionary<string, object> parameters)
```

Comment:
good catch. I didn't see the extension method. C# will bind to the instance method over the extension method.  However, we'll still need the extension method for backwards compat of already compiled code. I'll add an obsolete on the extension method.  

---
In reply to: [112491637](https://github.com/Azure/azure-webjobs-sdk/pull/1117#discussion_r112491637) [](ancestors = 112491637)